corehq/apps/tour/management/commands/mark_tour_as_seen.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib.auth.models import User
+from django.core.management.base import LabelCommand
+import sys
+from corehq.apps.tour.models import GuidedTours
+
+
+class Command(LabelCommand):
+    help = "marks a tour with tour_slug as seen for all existing users"
+    args = "tour_slug"
+
+    def handle(self, tour_slug, *args, **kwargs):
+        if len(args) > 0:
+            username = args[0]
+            confirm = raw_input("""MARK Tour {tour_slug} as SEEN for User {username}?
+    Type 'yes' to continue or 'no' to cancel: """.format(tour_slug=tour_slug, username=username))
+            if confirm == 'yes':
+                user = User.objects.filter(username=username).first()
+                GuidedTours.mark_as_seen(user, tour_slug)
+                print("Complete")
+        else:
+            confirm = raw_input("""MARK Tour {tour_slug} as SEEN for ALL USERS?
+    Type 'yes' to continue or 'no' to cancel: """.format(tour_slug=tour_slug))
```

Comment:
nit: consider putting these template strings in module-level constants to fix indentation here and make the indentation in the string easier to see.